queries/nu/highlights.scm

```diff
@@ -182,6 +182,14 @@ file_path: (val_string) @variable.parameter
     "e>"   "err>"
     "e+o>" "err+out>"
     "o+e>" "out+err>"
+    "o>>"   "out>>"
+    "e>>"   "err>>"
+    "e+o>>" "err+out>>"
+    "o+e>>" "out+err>>"
+    "o>|"   "out>|"
```

Comment:
although these appear in the code, `o>|` / `out>|`, i can't get them to do anything. maybe that's a bug? i'm not sure exactly.

Reply:
Does other options work for you?

<img width="105" alt="image" src="https://github.com/user-attachments/assets/8ce60c6f-8b52-4ba0-a5fd-523fe0eb8820" />

<img width="105" alt="image" src="https://github.com/user-attachments/assets/26d9ec34-86d9-47e0-a2b8-b5eff01e97c6" />

Seems to work on my computer though.

Reply:
I'm talking about the nushell functionality not working, not the tree-sitter-nu functionality not working.
![image](https://github.com/user-attachments/assets/d9b9c433-482b-4f93-a2aa-820a685e5069)

Reply:
I think we can remove those from the highlights.scm